README.md

```diff
@@ -172,10 +171,8 @@ apiVersion: openslo/v1
 kind: DataSource
 metadata:
   name: string
-  displayName: string # optional
```

Comment:
Why are we getting rid of `displayName` in `DataSource`?

Reply:
It's my mistake, I'm fixing